src/cases/controller/case.controller.test.js

```diff
@@ -36,37 +36,39 @@ function mockResponseToolkit(options = {}) {
 describe('Case Controller', () => {
   describe('listCases', () => {
     test('returns case list view with processed data when use case succeeds', async () => {
-      const mockCases = [
-        {
-          _id: 'case-1',
-          clientRef: 'client-ref-1',
-          code: 'case-code-1',
-          workflowCode: 'workflow-1',
-          currentStage: 'stage-1',
-          stages: ['stage-1', 'stage-2'],
-          createdAt: '2021-01-01T00:00:00.000Z',
-          submittedAt: '2021-01-15T10:30:00.000Z',
-          status: 'In Progress',
-          assignedUser: 'user-1'
-        },
-        {
-          _id: 'case-2',
-          clientRef: 'client-ref-2',
-          code: 'case-code-2',
-          workflowCode: 'workflow-2',
-          currentStage: 'stage-2',
-          stages: ['stage-1', 'stage-2'],
-          createdAt: '2021-02-01T00:00:00.000Z',
-          submittedAt: '2021-02-15T10:30:00.000Z',
-          status: 'Completed',
-          assignedUser: 'user-2'
-        }
-      ]
+      const mockCases = {
+        data: [
+          {
+            _id: 'case-1',
+            clientRef: 'client-ref-1',
+            code: 'case-code-1',
+            workflowCode: 'workflow-1',
+            currentStage: 'stage-1',
+            stages: ['stage-1', 'stage-2'],
+            createdAt: '2021-01-01T00:00:00.000Z',
+            submittedAt: '2021-01-15T10:30:00.000Z',
+            status: 'In Progress',
+            assignedUser: 'user-1'
+          },
+          {
+            _id: 'case-2',
+            clientRef: 'client-ref-2',
+            code: 'case-code-2',
+            workflowCode: 'workflow-2',
+            currentStage: 'stage-2',
+            stages: ['stage-1', 'stage-2'],
+            createdAt: '2021-02-01T00:00:00.000Z',
+            submittedAt: '2021-02-15T10:30:00.000Z',
+            status: 'Completed',
+            assignedUser: 'user-2'
+          }
+        ]
+      }
 
       const mockViewModel = {
         pageTitle: 'Cases',
         heading: 'Case Management',
-        cases: mockCases.map((c) => ({
+        cases: mockCases.data.map((c) => ({
```

Comment:
The API returns an array

```suggestion
        cases: mockCases.map((c) => ({
```